fleet/api/auth_test.go

```
@@ -0,0 +1,48 @@
+	_, ok = s.verify(tok)
+	require.False(t, ok, "expired")
+	other := newSessions([]byte("other"), time.Hour)
+	_, ok = other.verify(tok)
+	require.False(t, ok, "different secret")
+}
+
+func TestLimiter(t *testing.T) {
+	l := newLimiter(3, time.Minute)
+	now := time.Unix(0, 0)
+	l.now = func() time.Time { return now }
+	for i := 0; i < 3; i++ {
```

golangci-lint (2.13.2)

[error] for loop can be changed to use an integer range (Go 1.22+)

(intrange)